Author: Alessio Treglia <alessio@debian.org>
Origin: Debian
Description: Drop GCC options which are not supported on all architectures.
Bug: http://bugs.debian.org/584492
Forwarded: no
---
 src/CMakeLists.txt |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- yoshimi.orig/src/CMakeLists.txt
+++ yoshimi/src/CMakeLists.txt
@@ -74,7 +74,7 @@ set (BuildOptions_X86_64Core2
 )
 
 set (BuildOptionsBasic
-    "-O3 -march=pentium4 -msse -msse2 -mfpmath=sse -ffast-math -fomit-frame-pointer"
+    "-O3 -ffast-math -fomit-frame-pointer"
     CACHE STRING "X86 complier options"
 )
 
